.github/workflows/main.yml

```diff
@@ -0,0 +1,32 @@
+name: PeerPrep CI
+
+on:
+  push:
+    branches: 
+      - "master"
+  pull_request:
+    branches:
+      - "master"
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    name: Build
+    steps:
+    - uses: actions/checkout@v3
+    - name: Build the service
+      run: docker-compose up -d
+  
+  test:
+    runs-on: ubuntu-latest
+    name: Frontend Tests
+    needs: build
+    steps:
+    - uses: actions/checkout@v3
+    - name: Frontend
+      working-directory: ./frontend
+      run: |
+          npm install
+          npm install -g @angular/cli
+          xvfb-run -a ng test --watch=false --browsers=ChromeHeadless 
```

Comment:
Can I check if these are the commands you'd use to run the tests? 
Does it work for Jasmine tests in the backend too?

Reply:
Yea these were the commands to run the tests. I'm pretty sure you just paste the code you use to run the backend tests and it should work. The only reason why I had to do `xvfb-run -a ng test --watch=false --browsers=ChromeHeadless` was so that the tests can run on a "fake" chrome browser